package.json

```diff
@@ -69,6 +74,10 @@
 			}
 		],
 		"commands": [
+			{
+				"command": "helloworld-web-sample.helloWorld",
+				"title": "Hello World"
+			},
```

Comment:
This is just my ignorance talking: how is this command hooked up to produce output? (And is it needed?)

Reply:
Woops. Good spot. A mistake leftover from debugging. I'll remove it!